tests/unit/via/views/route_by_content_test.py

```diff
@@ -32,7 +33,7 @@ def test_we_call_third_parties_correctly(
         url = "http://example.com/path%2C?a=b"
         call_route_by_content(url, params={"other": "value"})
 
-        get_url_details.assert_called_once_with(url)
+        get_url_details.assert_called_once_with(url, Any.instance_of(EnvironHeaders))
```

Comment:
It would be nice to say `request.headers` here, but the request object is in a closure, and it would be bother to get it out.